### PR TITLE
Project Showcase Update

### PR DESCRIPTION
**Pull request by lzylili**

### CODE REVIEW

File: qosf.org/mentorship_cohort_2.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+    
+*Mentee*: **[Lisa Noorlander](https://github.com/LNoorl)** is a software engineer with a background in physics and mathematics. 
+
+*Mentor*: **[Julien Gacon](https://www.linkedin.com/in/julien-gacon/)** is a predoctoral researcher at IBM Quantum in Stefan Woerner’s group in Zurich. He’s currently working on application-aware circuit compilation and develops the open-source framework Qiskit. 
+
+*Links*: [Qiskit Contribution](https://github.com/Qiskit/qiskit-terra/pull/5657).
+
+---
+  
+##### **QuEST Open Source Simualtor Benchmark**
```
Comment (amirebrahimi) [inline qosf.org/mentorship_cohort_2.md:76]: Simulator

Comment (lzylili) [reply qosf.org/mentorship_cohort_2.md]: Thanks @amirebrahimi, fixed!

File: qosf.org/qc_mentorship.md [hunk 2]
```diff
@@ -48,9 +48,9 @@ If you're interested, you can learn more about the [story behind the program](ht
 
 ## Project Highlights
 
-Cohort one (March 2020 to June 2020): [mentorship projects]({{ 'cohort_1' | absolute_url }}).
+Cohort one (March 2020 to June 2020): [mentorship projects]({{ 'mentorship_cohort_1' | absolute_url }}).
 
-Cohort two (September 2020 to January 2021): mentorship projects will be shared on our [Twitter](https://twitter.com/qosfoundation) prior to being published on our website, as the cohort has just finished.
+Cohort two (September 2020 to January 2021): [mentorship projects]({{ 'mentorship_cohort_2' | absolute_url }}).
```
Comment (mstechly) [inline qosf.org/qc_mentorship.md:53]: Actually they started in October I think...

**Review by mstechly — APPROVED** [review]
One minor comment, overall looks great!

File: qosf.org/qc_mentorship.md (hunk 2 above)
Comment (lzylili) [reply qosf.org/qc_mentorship.md]: Fixed!

File: qosf.org/mentorship_cohort_1.md [hunk 3]
```diff
@@ -17,45 +17,29 @@ buttons:
...
-| Program applications released                           | February         | 
-| Applications review                                     | February/March   |
-| Project scope defined with a mini-proposal              | March            |
-| Mentees and mentors work on projects                    | April - June     |
-| Program wrap-up and open-source projects released!      | June             |
-
-### Project Showcase! 
-
-In total, there were 9 projects completed. However, there are still a couple more that were delayed by unforeseen circumstances! 
+This is a showcase of all the projects created during the first cohort of our Quantum Computing Mentorship program. Mentees and mentors worked on a open-source project, with the support of a community of Plike-minded people from all around the world. In total, there were 9 projects completed!
```
Comment (markf94) [inline qosf.org/mentorship_cohort_1.md:26]:
```suggestion
This is a showcase of all the projects created during the first cohort of our Quantum Computing Mentorship program. Mentees and mentors worked on a open-source project, with the support of a community of like-minded people from all around the world. In total, there were 9 projects completed! 
```

File: qosf.org/mentorship_cohort_2.md [hunk 4]
```diff
@@ -0,0 +1,312 @@
+---
+
+# Page settings
+layout: default
+keywords:
+comments: false
+
+# Hero section
+title: Quantum Computing Mentorship Program Cohort One Showcase
```
Comment (markf94) [inline qosf.org/mentorship_cohort_2.md:9]:
```suggestion
title: Quantum Computing Mentorship Program Cohort Two Showcase 
```

**Comment by markf94** [comment]
Thanks @lzylili for this contribution! It's a great showcase of all the projects!!

**Pull request merged**